projects/angular-auth-oidc-client/src/lib/utils/url/url.service.ts

```diff
@@ -488,6 +485,13 @@ export class UrlService {
     );
   }
 
+  private getCodeChallenge(config: OpenIdConfiguration): Observable<string> {
+    if (config.disablePkce) return of(null);
```

Comment:
Please spend the return an extra line and curly braces

```
if (config.disablePkce) {
   return of(null);
}
```

Reply:
done